golem_messages/message/tasks.py

```diff
@@ -17,7 +17,124 @@
 TASK_MSG_BASE = 2000
 
 
-class ComputeTaskDef(datastructures.ValidatingDict, datastructures.FrozenDict):
+class OUTPUT_FORMAT(datastructures.StringEnum):
+    PNG = enum.auto()
+    LXR = enum.auto()
```

Comment:
Isn't this list incomplete? What about JPEG or EXR?

Reply:
Actually it was. Now it is added and fixed LXR -> LXS